### PR TITLE
added page for trademark use guidelines

### PR DESCRIPTION
**Pull request by Franz-Kafka**

### CODE REVIEW

File: trademark_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Names derived from “Duck”, such as “duckling”, are also not allowed. Company names may not include “Duck”, unless otherwise agreed in writing.
+
+Package identifiers may include “duck”, but the full name used for the software package should follow the naming policy above.
+
+Written materials must refer to the project as “DuckDB” in the first and most prominent mentions. Materials from software vendors or software-related service providers must follow stricter guidelines, including using the full project name “DuckDB” in more locations, and proper trademark attribution on every page.
+
+Logos using the duck graphical and/or textual element, or otherwise derived from the DuckDB logos are also not allowed.
+
+Domain names relating to products or services based on the DuckDBcontaining “duck” are not permitted without written permission from the the DuckDB Foundation. Events, books, and merchandise have their own guidelines. Please contact us for details.
```
Comment (Mause) [inline trademark_guidelines.md:32]: Looks like some editing mistakes in this line

File: trademark_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Questions? Contact us
+
+The DuckDB Foundation may grant permission to apply DuckDB trademarks (including graphic logos) for merchandise that promotes the DuckDB project, software or community. Permission to apply DuckDB trademarks will ordinarily be denied for merchandise that disparages DuckDB software or projects or that would serve to detract from the value of DuckDB software. As a rule of thumb you may assume the following uses of DuckDB trademarks are probably infringing: 1) Confusingly similar software product names; 2) Software service offerings that are for anything other than official DuckDB software; and 3) Company names that may be associated in customer's minds with the DuckDB project, software or community. 
+
+We appreciate the community’s help in respecting these guidelines. If you have any questions about the DuckDB trademark guidelines or policy, or if you wish to request permission for any specific use case, please email us <address@duckdb.com> and CC <trademarks@duckdb.com>.
```
Comment (Mause) [inline trademark_guidelines.md:38]: Email addresses need to be updated

**Review by Mause — CHANGES_REQUESTED** [review]
Probably needs a section for whether or not these guidelines apply retroactively - there are many projects released in the past that violate these guidelines

**Comment by hannes** [comment]
will edit those after merging, thanks

File: faq.md [hunk 3]
```diff
@@ -15,6 +15,9 @@ Ducks are amazing animals. They can fly, walk and swim. They can also live off p
 ### Where do I find the DuckDB Logo?
 You can download the DuckDB Logo here: <br/> • Web: [png](/images/logo-dl/DuckDB_Logo.png) / [jpg](/images/logo-dl/DuckDB_Logo.jpg) <br/>  • Print: [svg](/images/logo-dl/DuckDB_Logo.svg) / [pdf](/images/logo-dl/DuckDB_Logo.pdf) <br/><br/>The DuckDB logo & website were designed by [Jonathan Auch](http://jonathan-auch.de) & [Max Wohlleber](https://maxwohlleber.de).
 
+### Where do I find DuckDB trademark use guidelines?
+You can find the gudelines for DUCKDB™ [here](https://duckdb.org/trademark_guidelines).
```
Comment (Mause) [inline faq.md:19]: Also some typos here I didn't notice before

**Pull request merged**